### ISSUE TITLE
(Re-)Publish Freactal intro on Hacker Noon?

### ISSUE THREAD

**Issue by agurtovoy** [FormidableLabs/freactal#91]
I'm not sure if you guys are looking for `freactal` to get more traction; please ignore this if you are not :).

If you are, however, I'd strongly suggest to republish your [awesome intro](https://formidable.com/blog/2017/infinite-state-composition-with-freactal/) on Medium and [submit](https://hackernoon.com/about) it to [Hacker Noon](https://hackernoon.com/).

**Comment by divmain** [comment]
Thanks @agurtovoy!  I switched to a new job shortly after publishing this project, and am only now picking it back up again.  I saw your fork [here](https://github.com/textpress/freactal).  A couple of questions for you:

1. Would you have any interest in incorporating those changes in this repo?
2. Would you have any interest in helping to maintain `freactal` formally?  I'll be spending time on it again, but would like to build up a team to help maintain is long term.

**Comment by agurtovoy** [comment]
@divmain Yes and yes. We all have our periods of busyness, so having a team of maintainers would be awesome. I'll start preparing PRs for some of our noncontroversial changes next week. Let me know what's your preferred medium for discussing the ones that are possibly questionable.

**Comment by bingomanatee** [comment]
How about gitter?
